src/components/WeekWeather.js

```diff
@@ -1,31 +1,28 @@
 import React from "react";
 import WeatherItem from "./WeatherItem";
-import { shadeColor } from "./../utils/shade-color";
+// import shadeColor from "../utils/shadeColor";
 
-import _ from "lodash";
-
-export default class WeekWeather extends React.Component {
-  constructor(props) {
-    super(props);
-  }
-
-  render() {
-    const color = this.props.color;
-
-    return <div className="week-container">
-      <div className="week-current-day">
-        <WeatherItem data={this.props.weekWeather[0]} />
-      </div>
-      <div className="week-all-days">
-        {_.map(this.props.weekWeather, (weather, i) => {
-          var style = {
-            backgroundColor: shadeColor(color, -(i + 1) / 20)
-          }
-          return <div key={i} className="week-one-day">
-            <WeatherItem theme={"small"} data={weather} />
-          </div>
-        })}
-      </div>
+const WeekWeather = ({color, weather, city, country}) => {
+  
+    return (
+      <div className="week-container">
+        <div className="week-current-day">
+          <WeatherItem data={weather[0]} />
+        </div>
+        <div className="week-all-days">
+          {weather.map((weatherOfDay, i) => {
+            // const style = {
+            //   backgroundColor: shadeColor(color, -(i + 1) / 20)
+            // }
+            return (
+              <div key={i} className="week-one-day">
+                <WeatherItem theme={"small"} data={weatherOfDay} city={city} country={country}/>
+              </div>
+            )
+          })}
```

Comment:
We can remove the explicit return and use `()` to wrap our div.
And also we can remove the commented part.